src/components/EmojiPicker/EmojiSkinToneList.js

```diff
@@ -20,75 +20,54 @@ const propTypes = {
     ...withLocalizePropTypes,
 };
 
-class EmojiSkinToneList extends Component {
-    constructor(props) {
-        super(props);
+function EmojiSkinToneList(props) {
```

Comment:
I didn't replicate the `componentDidMount`/`componentDidUpdate` functions, as I do not see the point of updating the `highlightedIndex`. We are later computing the `isHiglighted` prop as logical "or", which already checks if the index matches the value of `props.preferredSkinTone`.

Reply:
@szebniok `componentDidUpdate` was introduced to solve [this issue](https://github.com/Expensify/App/issues/14644). I cannot find the exact reason why `componentDidMount` was added but it might be required as well.

Reply:
Thanks for the detailed explanation @szebniok